app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CommentsController < ApplicationController
+  respond_to :html
+
+  expose(:comment, attributes: :comment_params)
+  expose(:comments) { Comment.page(params[:page]) }
+
+  def create
+    authorize comment
+
+    page = Page.find(params[:page_id])
+    comment.user = current_user
+    comment.page_id = page.id
+
+    if comment.save
+      redirect_to(Page.find(comment.page_id))
```

Comment:
redirect_to(Page.find(comment.page_id))

is the same as

redirect_to comment.page